app/src/main/java/com/example/poseexercise/views/fragment/WorkoutFragment.kt

```diff
@@ -84,25 +82,23 @@ class WorkOutFragment : Fragment() {
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View {
-
-        val view: View = inflater.inflate(R.layout.fragment_workout, container, false)
-
-        // Linking all button and controls
-        cameraFlipFAB = view.findViewById(R.id.facing_switch)
-        startButton = view.findViewById(R.id.button_start_exercise)
-        buttonCompleteExercise = view.findViewById(R.id.button_complete_exercise)
-        buttonCancelExercise = view.findViewById(R.id.button_cancel_exercise)
-        timerTextView = view.findViewById(R.id.timerTV)
-        timerRecordIcon = view.findViewById(R.id.timerRecIcon)
-
-        return view
+        return inflater.inflate(R.layout.fragment_workout, container, false)
     }
 
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        
+        // Linking all button and controls
         previewView = view.findViewById(R.id.preview_view)
         graphicOverlay = view.findViewById(R.id.graphic_overlay)
+        val cameraFlipFAB: FloatingActionButton = view.findViewById(R.id.facing_switch)
```

Comment:
I was running into error logcat of this so I moved the initialization of all the Button view to onViewCreated() instead of onCreateView()